### ISSUE TITLE
모델 학습 관련 문제 해결

### ISSUE THREAD

**Issue by IMBLOP** [IMBLOP/dialect-translator#1]
#### 1. 사투리-표준어의 내용이 다른 유효한 번역 문장 데이터만 사용
- 기존 데이터 230만 건에서 유효 데이터 33만 건만 학습 진행 (약 86% 감소)
- 학습 소요 시간 15H에서 2H30M으로 단축 (약 83% 단축)
#### 2. 기존 PreTrainedTokenizerFast에서 AutoTokenizer로 변경
- 데이터 및 학습 시간 단축으로 더 적합한 Tokenizer 사용
#### 3. Seq2SeqTrainer 모듈 추가
- 번역이나 요약 같은 시퀀스 생성 태스크에 맞게 전처리, 최적화, 하이퍼파라미터 조정 등을 보다 쉽게 할 수 있어 전체 학습 과정의 효율과 코드의 간결성을 높임
#### 4. 하이퍼파라미터 조정
- 기존 학습 버전 내역과 자료 등을 참고해 max_length, batch_size, learning_rate 수정
#### 5. Tokenizer Padding 동적으로 변경
- 기존 padding="max_length"(고정 길이)는 학습 문장의 길이가 짧아도 설정한 토큰 길이에 맞춰 padding
- padding=True(동적 패딩)는 한 배치 내에서 가장 긴 시퀀스 길이에 맞춰 padding해 실제 길이에 맞춰 최소한만 padding
